src/templates/pt/2019/base.html

```diff
@@ -0,0 +1,229 @@
+
+{% block part %}Parte{% endblock %}
+{% block chapter %}Capítulo{% endblock %}
+
+{% block start_exploring %}Comece a explorar{% endblock %}
+
+{% block menu_title %}Menu da Página{% endblock %}
+{% block open_the_menu %}Abra o menu{% endblock %}
+{% block close_the_menu %}Feche o menu{% endblock %}
+
+{% block index_nav_title %}Chapter table of contents{% endblock %}
+{% block index_title %}Index{% endblock %}
+{% block open_the_index %}Abrir índice de conteúdos{% endblock %}
+{% block close_the_index %}Fechar índice de conteúdos{% endblock %}
+
+{% block and %}e{% endblock %}
+
+{% block open %}Abrir{% endblock %}
+{% block close %}Fechar{% endblock %}
+
+{% macro show_description(id) %}Mostrar descrição da figura {{ id }}{% endmacro %}
+{% macro hide_description(id) %}Ocultar descrição da figura {{ id }}{% endmacro %}
+
+{% block written_by_before %}Escrito por{% endblock %}
+{% block reviewed_by_before %}Revisado por{% endblock %}
+{% block translated_by_before %}Traduzido por{% endblock %}
+{% block written_by_after %}{% endblock %}
+{% block reviewed_by_after %}{% endblock %}
+{% block translated_by_after %}{% endblock %}
+
+{% block author %}Autor(a){% endblock %}
+{% block authors %}Autores{% endblock %}
+
+{% macro onTwitter(twitterHandle) %}@{{twitterHandle}} no Twitter{% endmacro %}
+{% macro onGitHub(gitHubHandle) %}{{gitHubHandle}} no GitHub{% endmacro %}
+{% macro website(authorName) %}{{authorName}} website{% endmacro %}
+
+{% macro edition(year) %}{{ year }} Edição{% endmacro %}
+
+{% block footer_title %}Navegação no rodapé{% endblock %}
+{% block copyright %}© Web Almanac. Licenciado sob <a
+  href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/LICENSE">Apache 2.0</a>.{% endblock %}
+{% block accessibility_statement %}Declaração de acessibilidade{% endblock %}
+
+{% block contributors_description %}
+O Web Almanac Isso foi possível graças ao trabalho árduo da comunidade da web. {{ self.contributors() }} as pessoas
+ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção.
+{% endblock %}
+{% block contributors_link %}Veja os colaboradores{% endblock %}
+
+{% block methodology_stat_1_title %}Sites testados{% endblock %}
+{% block methodology_stat_1 %}5.8M{% endblock %}
+{% block methodology_stat_2_title %}Dados processados{% endblock %}
+{% block methodology_stat_2 %}20.9 TB{% endblock %}
+{% block methodology_description %}
+Salvo indicação em contrário, as métricas em todos os 20 capítulos do Web Almanac são originárias do conjunto de dados
+do HTTP Archive. O HTTP Archive é um projeto administrado pela comunidade que acompanha como a Web é criada desde 2010.
+Usando o WebPageTest e o Lighthouse, os metadados de cerca de 6 milhões de sites são testados mensalmente e incluídos em
+um banco de dados público do BigQuery para análise. O conjunto de dados de julho de 2019 foi usado como base para as
+métricas do Web Almanac. Para mais informações, consulte a página Metodologia.
```

Comment:
Combine these to keep line numbering same as English:

```suggestion
Salvo indicação em contrário, as métricas em todos os 20 capítulos do Web Almanac são originárias do conjunto de dados do HTTP Archive. O HTTP Archive é um projeto administrado pela comunidade que acompanha como a Web é criada desde 2010. Usando o WebPageTest e o Lighthouse, os metadados de cerca de 6 milhões de sites são testados mensalmente e incluídos em um banco de dados público do BigQuery para análise. O conjunto de dados de julho de 2019 foi usado como base para as métricas do Web Almanac. Para mais informações, consulte a página Metodologia.
```